src/components/Collapsible/index.js

```diff
@@ -25,7 +27,10 @@ const Collapsible = ({ nodeData: { children, options }, sectionDepth, ...rest })
     return findAllNestedAttribute(children, 'id');
   }, [children]);
 
-  const [open, setOpen] = useState(expanded ?? true);
+  const [open, setOpen] = useState(() => {
+    if (isOfflineDocsBuild || expanded) return true;
+    return expanded ?? true;
```

Comment:
(Nit) I'm wondering if we can simplify this to:

```suggestion
    if (isOfflineDocsBuild) return true;
    return expanded ?? true;
```

Checking `expanded` twice when it's false seems redundant, but maybe I'm missing something. Definitely an optional suggestion either way